src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/Remove-AzureRmRecoveryServicesAsrProtectionContainer.md

```diff
@@ -0,0 +1,108 @@
+---
+external help file: Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.dll-Help.xml
+Module Name: AzureRM.RecoveryServices.SiteRecovery
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.recoveryservices.siterecovery/remove-azurermrecoveryservicesasrprotectioncontainer
+schema: 2.0.0
+---
+
+# Remove-AzureRmRecoveryServicesAsrProtectionContainer
+
+## SYNOPSIS
+Deletes the specified Protection Container from its Fabric.
+
+## SYNTAX
+
+```
+Remove-AzureRmRecoveryServicesAsrProtectionContainer -InputObject <ASRProtectionContainer>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Remove-AzureRmRecoveryServicesAsrProtectionContainer cmdlet deletes the specified Azure Site Recovery Protection Container.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Remove-AzureRmRecoveryServicesAsrProtectionContainer -InputObject $protectionContainer
```

Comment:
@viverm it would be useful for the user to know where they get the value for the `$protectionContainer` object

Reply:
ok .. Added that